scalameter-core/src/main/scala/org/scalameter/keys.scala

```diff
@@ -4,10 +4,12 @@ package org.scalameter
 
 import java.util.Date
 import scala.collection._
+import org.scalameter.picklers.Pickler
+import org.scalameter.picklers.Implicits._
 
 
 
-class Key[T](val name: String)(implicit container: KeyContainer) extends Serializable {
+class Key[T: Pickler](val name: String)(implicit @transient container: KeyContainer) extends Serializable with PicklerBasedKey[T] {
```

Comment:
It looks like `container` became a part of the state class state - we don't have an annotation to prevent this.

In this case, I don't think we want `container` to be a part of the object state. Any reason not to make the `key` below a `val`?